src/SFA.DAS.Assessor.Functions/Functions/Ofqual/OfqualStager.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Assessor.Functions.Data;
+using SFA.DAS.Assessor.Functions.Domain.Entities.Ofqual;
+
+namespace SFA.DAS.Assessor.Functions.Functions.Ofqual
+{
+    public abstract class OfqualStager
+    {
+        private readonly IAssessorServiceRepository _assessorServiceRepository;
+        private readonly OfqualDataType _ofqualDataType;
+        private readonly string _ofqualDataTypeString;
+
+        protected OfqualStager(IAssessorServiceRepository assessorServiceRepository, OfqualDataType ofqualDataType)
+        {
+            _assessorServiceRepository = assessorServiceRepository;
+            _ofqualDataType = ofqualDataType;
+            _ofqualDataTypeString = ofqualDataType.ToString();
```

Comment:
A class level variable to avoid a couple of ToString()'s later on....  Just no

Reply:
Brain fart, it didn't even need to be ToStringed. Changed